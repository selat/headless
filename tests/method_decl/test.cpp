#include "proxy.hpp"

void Test::test() {
}
