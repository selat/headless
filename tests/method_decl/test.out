Please add the following headers:
class.hpp
    test

Please remove the following headers:
proxy.hpp
